### PR TITLE
Update Angular and Blazor label provider directives to match nimble-components

### PR DESCRIPTION
**Pull request by jattasNI**
# Pull Request

## 🤨 Rationale

#1747 has a maintenance task to ensure the default strings in Angular and Blazor directives match the default strings in nimble-components.

## 👩‍💻 Implementation

Some labels were only provided in nimble-components, not nimble-angular or Blazor. Copied those values over following existing patterns in directives and test files.

## 🧪 Testing

Updated tests but otherwise none since this is a mechanical change.

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts [hunk 1]
```diff
@@ -15,5 +15,10 @@ export class NimbleLabelProviderCoreWithDefaultsDirective {
         this.elementRef.nativeElement.popupDismiss = $localize`:Nimble popup - dismiss|:Close`;
         this.elementRef.nativeElement.numericDecrement = $localize`:Nimble numeric - decrement|:Decrement`;
         this.elementRef.nativeElement.numericIncrement = $localize`:Nimble numeric - increment|:Increment`;
+        this.elementRef.nativeElement.errorIcon = $localize`:Nimble icon - error|:Error`;
```
Comment (rajsite) [inline angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts:18]: The Blazor label providers are missing all these too. Wanna handle it in this PR (seems reasonable) or separate?

File: angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts [hunk 2]
```diff
@@ -15,5 +15,10 @@ export class NimbleLabelProviderCoreWithDefaultsDirective {
...
+        this.elementRef.nativeElement.warningIcon = $localize`:Nimble icon - warning|:Warning`;
+        this.elementRef.nativeElement.informationIcon = $localize`:Nimble icon - information|:Information`;
```
[the remark this answers is not in the document]
Comment (jattasNI) [reply angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts]: Renamed to `popupIconError` and similar. My goodness that touched a lot of files.

File: angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts (hunk 1 above)
Comment (jattasNI) [reply angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts]: Added Blazor and updated instructions in #1747 to include it too.

File: angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts [hunk 3]
```diff
@@ -15,5 +15,10 @@ export class NimbleLabelProviderCoreWithDefaultsDirective {
         this.elementRef.nativeElement.popupDismiss = $localize`:Nimble popup - dismiss|:Close`;
         this.elementRef.nativeElement.numericDecrement = $localize`:Nimble numeric - decrement|:Decrement`;
         this.elementRef.nativeElement.numericIncrement = $localize`:Nimble numeric - increment|:Increment`;
+        this.elementRef.nativeElement.popupIconError = $localize`:Nimble icon - error|:Error`;
+        this.elementRef.nativeElement.popupIconWarning = $localize`:Nimble icon - warning|:Warning`;
+        this.elementRef.nativeElement.popupIconInformation = $localize`:Nimble icon - information|:Information`;
```
Comment (rajsite) [inline angular-workspace/projects/ni/nimble-angular/label-provider/core/nimble-label-provider-core-with-defaults.directive.ts:20]: based on the other examples
```suggestion
        this.elementRef.nativeElement.popupIconError = $localize`:Nimble popup icon - error|:Error`;
        this.elementRef.nativeElement.popupIconWarning = $localize`:Nimble popup icon - warning|:Warning`;
        this.elementRef.nativeElement.popupIconInformation = $localize`:Nimble popup icon - information|:Information`;
```

File: angular-workspace/projects/ni/nimble-angular/label-provider/core/tests/nimble-label-provider-core-with-defaults.directive.spec.ts [hunk 4]
```diff
@@ -30,7 +30,12 @@ describe('Nimble LabelProviderCore withDefaults directive', () => {
         loadTranslations({
             [computeMsgId('Close', 'Nimble popup - dismiss')]: 'Translated close',
             [computeMsgId('Decrement', 'Nimble numeric - decrement')]: 'Translated decrement',
-            [computeMsgId('Increment', 'Nimble numeric - increment')]: 'Translated increment'
+            [computeMsgId('Increment', 'Nimble numeric - increment')]: 'Translated increment',
+            [computeMsgId('Error', 'Nimble icon - error')]: 'Translated error',
+            [computeMsgId('Warning', 'Nimble icon - warning')]: 'Translated warning',
+            [computeMsgId('Information', 'Nimble icon - information')]: 'Translated information',
```
Comment (rajsite) [inline angular-workspace/projects/ni/nimble-angular/label-provider/core/tests/nimble-label-provider-core-with-defaults.directive.spec.ts:36]:
```suggestion
            [computeMsgId('Error', 'Nimble popup icon - error')]: 'Translated error',
            [computeMsgId('Warning', 'Nimble popup icon - warning')]: 'Translated warning',
            [computeMsgId('Information', 'Nimble popup icon - information')]: 'Translated information',
```

File: change/@ni-nimble-angular-bb7e376e-ce72-4760-bb8d-f5775b8eefa8.json [hunk 5]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Update default values in label provider directives",
```
Comment (rajsite) [inline change/@ni-nimble-angular-bb7e376e-ce72-4760-bb8d-f5775b8eefa8.json:3]:
```suggestion
  "type": "minor",
  "comment": "Update default values in label provider directives and expose additional label provider properties",
```

**Review by rajsite — APPROVED** [review]

**Comment by jattasNI** [comment]
Bypassed owner review policy because Blazor owners @msmithNI and @atmgrifter00 are out of office and Blazor changes were mechanical. FYI @rajsite

**Pull request merged**